.github/workflows/build_cmake.yml

```diff
@@ -25,7 +25,7 @@ jobs:
             name: "Ubuntu AppImage", artifact: "Avogadro2.AppImage",
             os: ubuntu-16.04,
             cc: "gcc", cxx: "g++",
-            cmake_flags: "-G Ninja -DENABLE_TESTING=ON ",
+            cmake_flags: "-G Ninja -DPYTHON_WHEEL_BUILD=ON ",
```

Comment:
It really would be preferable to not use this outside of the wheel build, so I named it very specifically. Ubuntu 20.04 is out, I feel like the oldest we should aim at is 18.04 and that can handle C++17 maybe...